### ISSUE TITLE
improving navigation to associated resources

### ISSUE THREAD

**Issue by eroux** [buda-base/public-digital-library#226]
I think in the menu on the left there could be something like:

```
Browse associated resources
   persons(~9)
   places(~3)
```

etc. in the same style at the rest, leading to the usuall search on associated resources

**Comment by JannTibetan** [comment]
Interestingly, the "associated places" data for Person records in included on BUDA but is part of the overall detailed record and not shunted off to a different tab (which is good).

**Comment by eroux** [comment]
yes, in most cases the data can be inlined... it's a change from tbrc.org and thus requires some getting used to, but I think it makes the UI a bit more simple

**Comment by JannTibetan** [comment]
I agree. It's better for information such as this to be integrated into a single detailed record.
